api_server.go

```diff
@@ -68,39 +70,48 @@ func logRequest(handlerFunc APIHandlerFunc, r *http.Request) ([]byte, int) {
 	case code >= 500:
 		logger.WithFields(fields).Error("request failed")
 	case 400 <= code && code < 500:
-		logger.WithFields(fields).Error("request failed")
+		logger.WithFields(fields).Error("invalid request")
 	default:
 		logger.WithFields(fields).Info("request completed")
 	}
 	return body, code
 }
 
-func (x *ApiServer) MusicPDFsIndex(r *http.Request) ([]byte, int) {
+func (x *ApiServer) SheetsIndex(r *http.Request) ([]byte, int) {
 	// only accept get
 	if r.Method != http.MethodGet {
 		return nil, http.StatusMethodNotAllowed
 	}
 
-	objects := x.ListObjects(MusicPdfsBucketName)
-	pdfs := make([]MusicPDFMeta, 0, len(objects))
+	type tableRow struct {
+		Sheet
+		Link string `json:"link"`
+	}
+
+	objects := x.ListObjects(SheetsBucketName)
```

Comment:
Consider modifying `ListObjects` to take a `prefix` argument, to reduce the number of objects to filter through. This isn't really needed right now, though.

Reply:
I'll add a QuerySQL() method next.